### PR TITLE
Add draft content to the Type scale and pairing guideline page

### PR DESCRIPTION
**Pull request by JennySanchez**
Page structure is WIP as well.

### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/pages/guidelines/type-pairing-and-spacing.mdx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+</Column>
+</Row>
+
+**Tokens**
+
+| `Type token` | `Breakpoint` | `info` |
+| --- | --- | --- |
+|`$expressive-heading-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.5em <br/>Weight: 600 / Semi-Bold <br/>Letter-spacing: 0px |
+|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 22px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```
Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:44]:
```suggestion
|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```

File: src/pages/guidelines/type-pairing-and-spacing.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+</Column>
+</Row>
+
+**Tokens**
+
+| `Type token` | `Breakpoint` | `info` |
+| --- | --- | --- |
+|`$expressive-heading-04`| Large |Type: IBM Plex Sans <br/>Size: 28px / 1.75rem <br/>Line-height: 36px / 2.25em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
+|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 22px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```
Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:100]:
```suggestion
|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```

File: src/pages/guidelines/type-pairing-and-spacing.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+<AnchorLink>Designing with type parings and spacings</AnchorLink>
+<AnchorLink>Support</AnchorLink>
+
+</AnchorLinks>
+
+## Designing with type parings and spacings
+
+Type parings and spacings support different levels of architecture or areas that require varying typographic hierarchies and are the keys to making the best IBM.com experiences through typography successfully.
+
+Below are some known type parings and spacings when designing for IBM.com to ensure you create clear hierarchies, proper information organization, and the best experience to guide users through the IBM.com experience.
```
Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:23]: Should we rephrase to "Below are some common type pairings" ? Thinking that these are the ones we have built into the components, "known" just felt a little odd when I read it.

Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:23]:
```suggestion
Below are some known type pairings and spacings when designing for IBM.com to ensure you create clear hierarchies, proper information organization, and the best experience to guide users through the IBM.com journey.
```

Comment (oliviaflory) [reply src/pages/guidelines/type-pairing-and-spacing.mdx:23]: Seemed repetitive to say "experience" twice ?

File: src/pages/guidelines/type-pairing-and-spacing.mdx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+</Column>
+</Row>
+
+**Tokens**
+
+| `Type token` | `Breakpoint` | `info` |
+| --- | --- | --- |
+|`$expressive-heading-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.5em <br/>Weight: 600 / Semi-Bold <br/>Letter-spacing: 0px |
+|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 22px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```
Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:128]:
```suggestion
|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```

**Comment by oliviaflory** [comment]
@JennySanchez adding some general comment here too

1. Would we consider making the way we write the type tokens consistent? In the first sentence they're capitalized, the other instances are lower case (I think lower case is correct)

I've seen Carbon use the inline code snippet when citing type tokens so the first sentence could be:
"Example of `$expressive-heading-02` paired with `$body-long-02` at the large breakpoint."

Screen Shot 2021-05-18 at 8 04 26 PM

---


2. We don't mention the space between any of the pairings within the documentation, other than the image. I had thought it might go within the chart when we reviewed as a team before. Wonder if that's worth finding somewhere to write it out?

We could also consider describing it in the sentence about the image

Content block pairs `$expressive-heading-05` and `$expressive-heading-03` with 32px of space between the two type styles, the example below shows the large breakpoint.

Screen Shot 2021-05-18 at 8 16 49 PM

File: src/pages/guidelines/type-pairing-and-spacing.mdx (hunk 3 above)
Comment (JennySanchez) [reply src/pages/guidelines/type-pairing-and-spacing.mdx]: Good point.

File: src/pages/guidelines/type-pairing-and-spacing.mdx [hunk 5]
```diff
@@ -0,0 +1,142 @@
+---
+title: Type pairing and spacing
+description: Typography can help create clear hierarchies, organize information, and guide users through a product or experience. Type parings and spacings support different levels of architecture or areas that require varying typographic hierarchies and are the keys to making the best IBM.com experiences through typography successfully.
+---
+
+<PageDescription>
+
+Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before dive into creating an IBM.com website page.
```
Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:8]:
```suggestion
Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before diving into creating an IBM.com experience.
```

Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:8]:
```suggestion
Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before diving into creating an IBM.com experience.
```

File: src/pages/guidelines/type-pairing-and-spacing.mdx [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+<AnchorLinks>
+
+<AnchorLink>Designing with type parings and spacings</AnchorLink>
+<AnchorLink>Support</AnchorLink>
+
+</AnchorLinks>
+
+## Designing with type parings and spacings
+
+Type parings and spacings support different levels of architecture or areas that require varying typographic hierarchies and are the keys to making the best IBM.com experiences through typography successfully.
```
Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:21]:
```suggestion
Consistent type pairing and spacing can support different levels of architecture or varying levels of hierarchy, which are the keys to making the best IBM.com experience.
```

File: src/pages/guidelines/type-pairing-and-spacing.mdx [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+| Breakpoint | `max` | `xlg` | `lg` | `md` | `sm` | 
+| --- | --- | --- | --- | --- | --- |
+| Padding top | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px |
+| Padding bottom | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px |
+
+<br />
+
+## Support
+
+For any reason, if the pairings or spacings listed above do not meet your needs, please contact the Digital Design System team via the [#carbon-for-ibm-dotcom slack channel](https://cognitive-app.slack.com/archives/C2PLX8GQ6).
```
Comment (oliviaflory) [inline src/pages/guidelines/type-pairing-and-spacing.mdx:142]:
```suggestion
If for any reason, the type pairings or spacings listed above do not meet your needs, please contact the Digital Design System team via the [#carbon-for-ibm-dotcom slack channel](https://cognitive-app.slack.com/archives/C2PLX8GQ6). 
```

**Review by oliviaflory — CHANGES_REQUESTED** [review]

**Review by oliviaflory — APPROVED** [review]
LGTM

**Pull request merged**